<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Mandelbrot-Qt</string>
  </property>
  <widget class="ViewerWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExample">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuick_save"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste_coords"/>
    <addaction name="separator"/>
    <addaction name="actionDeveloper_mode"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionAuto_iterations"/>
    <addaction name="actionIncrease_iterations"/>
    <addaction name="actionDecrease_iterations"/>
    <addaction name="separator"/>
    <addaction name="actionSingle_threaded"/>
    <addaction name="actionMultithreading"/>
    <addaction name="actionMax_threads"/>
    <addaction name="separator"/>
    <addaction name="actionShading"/>
    <addaction name="actionAutomatic_gradient"/>
    <addaction name="actionRandomize_palette"/>
    <addaction name="actionScale_palette"/>
    <addaction name="separator"/>
    <addaction name="actionOversampling"/>
    <addaction name="actionControl_panel"/>
    <addaction name="separator"/>
    <addaction name="actionShow_orbits"/>
   </widget>
   <widget class="QMenu" name="menuFractal">
    <property name="title">
     <string>Fractal</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="actionGoTo"/>
    <addaction name="actionAutozoom"/>
    <addaction name="actionAnimate_to_here"/>
    <addaction name="actionZoom_continuously"/>
    <addaction name="separator"/>
    <addaction name="actionQuality_animation"/>
    <addaction name="actionFast_animation"/>
    <addaction name="actionVery_fast_animation"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks_2">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionAdd_bookmark"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuExample"/>
   <addaction name="menuOptions"/>
   <addaction name="menuFractal"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuBookmarks_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSpeed">
   <property name="text">
    <string>Speed</string>
   </property>
  </action>
  <action name="actionQuality">
   <property name="text">
    <string>Quality</string>
   </property>
  </action>
  <action name="actionExport_image">
   <property name="text">
    <string>Export image</string>
   </property>
  </action>
  <action name="actionSave_position">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C, Meta+C</string>
   </property>
  </action>
  <action name="actionCopy_image">
   <property name="text">
    <string>Copy image</string>
   </property>
   <property name="shortcut">
    <string>Meta+C</string>
   </property>
  </action>
  <action name="actionMandelbrot">
   <property name="text">
    <string>Mandelbrot</string>
   </property>
  </action>
  <action name="actionIncrease_iterations">
   <property name="text">
    <string>Increase depth</string>
   </property>
  </action>
  <action name="actionDecrease_iterations">
   <property name="text">
    <string>Decrease depth</string>
   </property>
  </action>
  <action name="actionMake_movie">
   <property name="text">
    <string>Make movie...</string>
   </property>
  </action>
  <action name="actionColours">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Colours...</string>
   </property>
  </action>
  <action name="actionRandomize_palette">
   <property name="text">
    <string>Recolour palette</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionThreads">
   <property name="text">
    <string>Threads</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGoTo">
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAuto_iterations">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic depth</string>
   </property>
  </action>
  <action name="actionMultithreading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Multithreaded</string>
   </property>
  </action>
  <action name="actionQuick_save">
   <property name="text">
    <string>Quick save</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionScale_palette">
   <property name="text">
    <string>Enhance palette</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in at cursor</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>Center</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionAutozoom">
   <property name="text">
    <string>Navigate randomly</string>
   </property>
  </action>
  <action name="actionAnimate_to_here">
   <property name="text">
    <string>Animate to here</string>
   </property>
  </action>
  <action name="actionQuality_animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quality animation</string>
   </property>
  </action>
  <action name="actionFast_animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Smooth animation</string>
   </property>
  </action>
  <action name="actionVery_fast_animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fast animation</string>
   </property>
  </action>
  <action name="actionSingle_threaded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Single threaded</string>
   </property>
  </action>
  <action name="actionFour_threads">
   <property name="text">
    <string>Four threads</string>
   </property>
  </action>
  <action name="actionMax_threads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Maximum threads</string>
   </property>
  </action>
  <action name="actionZoom_continuously">
   <property name="text">
    <string>Zoom continuously at cursor</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionOversampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High resolution</string>
   </property>
  </action>
  <action name="actionPaste_coords">
   <property name="text">
    <string>Paste coordinates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAdd_bookmark">
   <property name="text">
    <string>Add bookmark...</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDeveloper_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Developer mode</string>
   </property>
  </action>
  <action name="actionAutomatic_gradient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic gradient</string>
   </property>
  </action>
  <action name="actionShading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shading</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionControl_panel">
   <property name="text">
    <string>Colour controls...</string>
   </property>
  </action>
  <action name="actionShow_orbits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show orbits</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerWidget</class>
   <extends>QWidget</extends>
   <header>ViewerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
